src/main/backend.ts

```diff
@@ -86,6 +86,13 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
    * @inheritDoc
    */
   public sendEvent(event: Event): void {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    if ((event as any).event && (event as any).event.__INTERNAL_MINIDUMP) {
+      // We are not 100% sure when such an event is happening, regardsless we def don't want to send it
```

Comment:
```suggestion
      // We are not 100% sure when such an event is happening, regardless we def don't want to send it
```

If there's some sort of debug mode, it would be great to print the stack trace or some other information that allows us to trace it back to its origin.